x/service/types/msg_update_params.go

```diff
@@ -0,0 +1,21 @@
+package types
```

Comment:
Filename seem to be inconsistent with the other `types/message_XXX`. Shouldn't we rename it to `message_update_params.go`?